tui-markdown/src/snapshots/tui_markdown__tests__code-2.snap

```diff
@@ -0,0 +1,352 @@
+---
+source: tui-markdown/src/lib.rs
+expression: "from_str(indoc!\n{\n    \"\n                ```rust\n                fn main() {\n                    println!(\\\"Hello, world!\\\");\n                }\n                ```\n            \"\n})"
```

Comment:
Is it always the expression inside the macro? Maybe it's an idea to assign variables with descriptive names.

Reply:
Seems to work by assigning to a variable first, I've also changed the blocks slightly so it's a bit easier when reviewing

Reply:
In theory this could also be manually written in yaml as something like
```yaml
expression: |
    from_str(indoc!{"
    ...

From some naive testing, insta doesn't seem to care too much about the expression regardless, so the indoc parts are probably unnecessary there either (their effect is just to remove the initial indentation in the rust code). This is not a big deal however.